FlagsmithClient/src/main/java/com/flagsmith/internal/FlagsmithRetrofitService.kt

```diff
@@ -3,26 +3,43 @@ package com.flagsmith.internal;
 import android.content.Context
 import android.util.Log
 import com.flagsmith.FlagsmithCacheConfig
+import com.flagsmith.entities.FeatureStatePutBody
 import com.flagsmith.entities.Flag
 import com.flagsmith.entities.IdentityFlagsAndTraits
 import com.flagsmith.entities.TraitWithIdentity
+import okhttp3.Cache
 import okhttp3.Interceptor
 import okhttp3.OkHttpClient
 import retrofit2.*
 import retrofit2.converter.gson.GsonConverterFactory
 import retrofit2.http.Body
 import retrofit2.http.GET
+import retrofit2.http.Header
 import retrofit2.http.POST
+import retrofit2.http.PUT
+import retrofit2.http.Path
 import retrofit2.http.Query
 
-interface FlagsmithRetrofitService {
+internal interface FlagsmithRetrofitService {
 
     @GET("identities/")
     fun getIdentityFlagsAndTraits(@Query("identifier") identity: String) : Call<IdentityFlagsAndTraits>
 
     @GET("flags/")
     fun getFlags() : Call<List<Flag>>
 
+    @GET("environments/{environmentKey}/featurestates/{featureStateId}/")
+    fun getFeatureStates(@Header("authorization") authToken:String,
+                         @Path("featureStateId") featureStateId: String,
+                         @Path("environmentKey") environmentKey: String,
+                         @Query("feature_name") featureName: String) : Call<String>
+
+    @PUT("environments/{environmentKey}/featurestates/{featureStateId}/")
+    fun setFeatureStates(@Header("authorization") authToken:String,
+                         @Path("featureStateId") featureStateId: String,
+                         @Path("environmentKey") environmentKey: String,
+                         @Body body: FeatureStatePutBody) : Call<Unit>
```

Comment:
Do these need to be set here on the `FlagsmithRetrofitService` or should we extend the class in the `/src/test` directories?

Reply:
Now that it's internal I don't suppose it really matters but would make sense to move it to the tests if we expose it

Reply:
I would prefer any code that is only used in testing to live in the `test` dirs.

Reply:
This should be sorted now